.github/workflows/build.yaml

```
@@ -65,6 +58,19 @@ on:
+          - "household"
+          - "individual"
+          - "project"
+          - "referralmanagement"
+          - "stock"
+          - "product"
+          - "project-factory"
+          - "transformer"
+          - "beneficiary-idgen"
+          - "egov-hrms"
+          - "pgr-services"
+          - "service-request"
```

actionlint (1.7.7)

option "service-request" is duplicated in options of "service" input

(events)